key.core/src/main/antlr4/KeYParser.g4

```diff
@@ -360,7 +361,7 @@ atom_prefix:
     update_term
   | substitution_term
   | locset_term
-  | cast_term
+  //| cast_term
```

Comment:
We should discuss, if the `cast_term` is needed. 

`cast_term` allows you to write Java-like cast expression `(int) x` in KeY. Drawback is the required lookahead: The parser needs to check the whole expression inside the parenthesis, until it recognise the missing operator after the right paren.

On the other side, we also use `<sortId>::<expr>` (in `accessterm`) for casting.

Reply:
Do you mean `<sortId>::cast(<expr>)`?

That would be breaking change that needs to be discussed.

Reply:
Yes. But at least `cast_term` is not ambigue.

Reply:
I'd like to keep the syntax.